docs/user/disconnected-cluster.md

```diff
@@ -0,0 +1,187 @@
+
+### see list of images to use
+
+```shell
+helm show values oci://quay.io/flightctl/charts/flightctl --version=0.4.0-rc3
+```
+
+### mirror flightctl images
+
+```shell
+oc image mirror -a combined-secret.json quay.io/flightctl/flightctl-worker:0.4.0-rc3  <registry-hostname>:5000/flightctl/flightctl-worker:0.4.0-rc3
+
+oc image mirror -a combined-secret.json quay.io/flightctl/flightctl-periodic:0.4.0-rc3  <registry-hostname>:5000/flightctl/flightctl-periodic:0.4.0-rc3
+
+oc image mirror -a combined-secret.json quay.io/flightctl/flightctl-ui:0.4.0-rc3  <registry-hostname>:5000/flightctl/flightctl-ui:0.4.0-rc3
+
+oc image mirror -a combined-secret.json quay.io/flightctl/flightctl-api:0.4.0-rc3  <registry-hostname>:5000/flightctl/flightctl-api:0.4.0-rc3
+```
+
+### mirror `keycloak`, `redis` and `postgresql` (please note: format of `keycloak` in `values.yaml` is different from others)
+
+```shell
+oc image mirror -a combined-secret.json quay.io/keycloak/keycloak:25.0.1 <registry-hostname>:5000/keycloak/keycloak:25.0.1
+
+oc image mirror -a combined-secret.json quay.io/sclorg/redis-7-c9s:latest <registry-hostname>:5000/sclorg/redis-7-c9s
+```
+
+> 📌 Note: PostgreSQL Version Upgrade will occur in `0.5.0`, so `postgresql-12-c8s` will not be required
+
+```shell
+oc image mirror -a combined-secret.json quay.io/sclorg/postgresql-12-c8s:latest <registry-hostname>:5000/sclorg/postgresql-12-c8s
+
+oc image mirror -a combined-secret.json quay.io/sclorg/postgresql-16-c9s:latest <registry-hostname>:5000/sclorg/postgresql-16-c9s
+```
+
+### `sha256`
+
+Look for (usually the last line)
+
+```kubernetes helm
+sha256:b33dedfeb245199a03122505d35d9c75ae01d77ffe39e0950ed98b0d02ecdf64 <registry-hostname>:5000/flightctl/flightctl-kv:latest
+```
+
+and add it to `values.yaml` ([See Example](#an-example-of-valuesyaml)). Please note
+
+- `@ha256` must be placed at the end of image name
+- `sha256`'s value must be placed under `tag`
+
+## Install flightctl
+
+### clear previous installation if required
+
+```shell
+helm uninstall flightctl -n flightctl
+
+kubectl delete pvc --all -n flightctl
+```
+
+### install
+
+```shell
+helm upgrade --install --version=0.4.0-rc3  --namespace flightctl --create-namespace flightctl oci://quay.io/flightctl/charts/flightctl --values values.yaml
+```
+
+## Troubleshooting
+
+### watch `pods` starting up
+
+Pod startup failures may occur multiple times due to component interdependencies.
+This is expected behavior as some components need to wait for others to be ready.
+
+```shell
+oc get pods -n flightctl -w
+```
+
+### get description
+
+```shell
+oc describe pods flightctl-api-546ff9d5b9-ccf8c -n flightctl
+```
+
+### List all pods excluding those with 'run' or 'comp' in their status (typically Running or Completed)
+
+```shell
+oc get pods -A |grep -vEi "run|comp"
+```
+
+### An example of `values.yaml`
+
+```yaml
+kv:
+  enabled: true
+  image:
+    image: "<registry-hostname>:5000/sclorg/redis-7-c9s@sha256"
```

Comment:
why  not the whole line with the image.  leave tag empty.

Reply:
with an empty tag it will add :whatever at the end